data-esnext.js

```diff
@@ -716,21 +716,23 @@ exports.tests = [
   name: 'class decorators',
   category: STAGE2,
   significance: 'medium',
-  spec: 'https://github.com/wycats/javascript-decorators',
+  spec: 'https://tc39.github.io/proposal-decorators/',
   exec: function(){/*
     class A {
       @nonconf
       get B() {}
     }
-    function nonconf(target, name, descriptor) {
-      descriptor.configurable = false;
-      return descriptor;
+    function nonconf(target) {
+      target.descriptor.configurable = false;
+      target.key = 'C';
+      return target;
     }
-    return Object.getOwnPropertyDescriptor(A.prototype, "B").configurable === false;
+    return Object.getOwnPropertyDescriptor(A.prototype, 'B') === undefined &&
+           Object.getOwnPropertyDescriptor(A.prototype, 'C').configurable === false;
   */},
   res: {
-    babel: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Babel 6 still has no official support decorators, but you can use <a href='https://github.com/loganfsmyth/regenerator-plugin-transform-decorators-legacy'>this plugin</a>."},
-    typescript: true,
+    babel: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Stage 2 decorators are in progress, see <a href='https://github.com/babel/babel/issues/2645'>babel/babel#2645</a>."},
```

Comment:
My main reason for creating this PR is that I keep running into people who find the legacy transform, assume that it's basically the same thing, want to create libraries based on the interface, etc.. So I'd rather not promote it, not even in a footnote. The rollout of the revised decorators will be painful enough already without even more users of the legacy transform.

Fair point about TypeScript. Maybe the footnote could be removed because afaict there's no public signals yet that TypeScript wants to implement this proposal anytime soon.